bin/update_ati_analytics.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e;
+
+function error_message {
+
+    if [[ $1 ]]; then
+
+        printf "\n\033[1;33m${1}\033[0;0m\n\n";
+        exit;
+
+    fi
+
+}
+
+function success_message {
+
+    if [[ $1 ]]; then
+
+        printf "\n\033[0;36m\033[1m${1}\033[0;0m\n\n";
+
+    fi
+
+}
+
+config_file=${REGISTRY_INI};
+
+if [[ -z "${config_file}" ]]; then
+
+    error_message 'REGISTRY_INI environment variable not set.';
+
+fi
+
+if [[ -z "$(command -v ckanapi)" ]]; then
+
+    error_message 'ckanapi command not found. Is it accessible in the PATH variable?';
+
+fi
+
+if [[ ! -d "/opt/tbs/tmp" ]]; then
+
+    error_message '/opt/tbs/tmp directory does not exist.';
+
+fi
+
+generated_file='/opt/tbs/ckan/smb_portal/portal/public/ati-informal-requests-analytics.csv';
+
+if [[ ! -f "${generated_file}" ]]; then
+
+    generated_file='/opt/tbs/ckan/smb/portal/public/ati-informal-requests-analytics.csv';
+
+    if [[ ! -f "${generated_file}" ]]; then
+
+        error_message '/opt/tbs/ckan/[smb_portal|smb]/portal/public/ati-informal-requests-analytics.csv does not exist.';
+
+    fi
+
+fi
+
+tmp_file='/opt/tbs/tmp/ati-informal-requests-analytics.csv';
```

Comment:
`mktemp` can securely create unique file names so that you don't have to worry about cases where the script is run twice and one truncates the file from the other.

Reply:
@wardi  https://github.com/open-data/ckanext-canada/pull/1327/commits/5aa63272e754bd33fa7339742eef67c293d9dbda

I used temp directories instead of files so that the resource in ckan would keep the file name `ati-informal-requests-analytics.csv`